### PR TITLE
Add MultiLaunch test

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

**Comment by IntelOrca** [comment]
`tests.vcxproj` needs updating

File: test/tests/MultiLaunch.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		ASSERT_TRUE(initialised);
+
+		game_load_sv6_path(path.c_str());
+
+		// Check ride count to check load was successful
+		ASSERT_EQ(gRideCount, 134);
+
+		delete context;
+	}
+	SUCCEED();
```
Comment (janisozaur) [inline test/tests/MultiLaunch.cpp]: Yeah, I noticed too late, sorry. Fixed.

**Comment by janisozaur** [comment]
@IntelOrca is AppVeyor running this test? How do I prevent it from doing so?

**Comment by IntelOrca** [comment]
https://github.com/OpenRCT2/OpenRCT2/blob/develop/openrct2.proj#L189

**Comment by janisozaur** [comment]
@IntelOrca it works with your improvements, many thanks!

**Pull request merged**